### PR TITLE
check for duplicate brick entry

### PR DESCRIPTION
**Pull request by Madhu-1**
check for the duplicate entry of the brick in volume create.

**example cli command for volume create** 
cli volume create testvolume node1:/brick1 node1:/brick1 --force

check for the duplicate entry of the brick in volume expand (provided brick entry already present in the volume)

**example cli command for volume add-brick** 
cli add volume add-brick testvolume node1:/brick1 --force (brick1 already present in volume)

for both example server should return an error message.

### CODE REVIEW

**Comment by Madhu-1** [comment]
fix for #723 

@prashanthpai  @kshlm @aravindavk  , please review

File: glusterd2/commands/volumes/volume-expand.go [hunk 1]
```diff
@@ -47,6 +48,26 @@ func volumeExpandHandler(w http.ResponseWriter, r *http.Request) {
...
+	volinfo, err := volume.GetVolume(volname)
+	if err != nil {
+		restutils.SendHTTPError(ctx, w, http.StatusInternalServerError, err)
+		return
+	}
+
+	for index := range volinfo.Subvols {
+		for _, brick := range volinfo.Subvols[index].Bricks {
+
+			for _, b := range req.Bricks {
```
Comment (aravindavk) [inline glusterd2/commands/volumes/volume-expand.go]: duplicate entries need to be checked within `req.Bricks` also.

Comment (Madhu-1) [reply glusterd2/commands/volumes/volume-expand.go]: @aravindavk added check

**Comment by Madhu-1** [comment]
do not merge, I think need to add logic to extract node UUID if node IP or hostname is passed in the brick entry.

**Comment by Madhu-1** [comment]
@prashanthpai  @kshlm  please review

**Pull request merged**